apps/challenges/views.py

```diff
@@ -937,7 +937,7 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
     except:
         try:
             if response_data:
-                response_data = {'error': response_data.values()[0]}
+                response_data = {'error': list(response_data.values())[0][0]}
```

Comment:
@KhalidRmb Looks like an issue with indexing. You are picking the first char of the message string. Please check.
```suggestion
                response_data = {'error': list(response_data.values())[0]}
```

Reply:
@rishabhbhardwaj Actually in this particular case the response_data is storing serializer.errors which itself is a dictionary. (Refer https://github.com/encode/django-rest-framework/blob/master/rest_framework/serializers.py). And that's why I'm accessing the 0 element twice.

Reply:
Noted. I'll make the change.